src/actions/profileActions/profileActionCreator.js

```diff
@@ -0,0 +1,33 @@
+import ACTION_TYPE from '../authActions/actionTypes';
```

Comment:
This file and `src/actions/profileActions/profileActions.js` should all be one file

Reply:
Resolved